components/builder-api-client/src/builder.rs

```diff
@@ -902,8 +948,9 @@ impl BuilderAPIProvider for BuilderAPIClient {
         let ident = pa.ident()?;
         let target = pa.target()?;
 
-        let mut file =
-            File::open(&pa.path).map_err(|e| Error::PackageReadError(pa.path.clone(), e))?;
+        debug!("Uploading package {}, target {}", ident, target);
```

Comment:
I am _soooooo_ looking forward to a day when target and ident can be part of a single unified identifier.

Reply:
That's part of "v1" right? :)

Reply:
It'd be real nice!